src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -227,3 +227,16 @@ def get_arg_list(op):
 def should_disable_connection_verify():
     import os
     return bool(os.environ.get(DISABLE_VERIFY_VARIABLE_NAME))
+
+
+def poller_classes():
+    from msrestazure.azure_operation import AzureOperationPoller
+    from msrest.polling.poller import LROPoller
+    return (AzureOperationPoller, LROPoller)
+
+
+def no_wait_params(is_no_wait):
+    """ Handle the parameters required for no-wait support in Autorest 2 & 3 """
+    if is_no_wait:
+        return {'raw': True, 'polling': False}
+    return {'raw': False, 'polling': True}
```

Comment:
If it's not "no wait", I feel like you should return an empty dict and let the default behavior does its job. You might create a parameter conflict here, when the custom implementation of `generatevpn` will be passed as `polling.`